shared/translations/de_DE.json

```diff
@@ -0,0 +1,19 @@
+{
+    "Home": "Startseite",
+    "Recently updated": "Zuletzt geändert",
+    "Recently viewed": "Zuletzt gesehen",
+    "Created by me": "Von mir erstellt",
+    "Profile saved": "Profil gespeichert",
+    "Profile picture updated": "Profilfoto wurde aktualisiert.",
+    "Unable to upload new avatar": "Neuer Avatar kann nicht hochgeladen werden.",
+    "Profile": "Profil",
+    "Photo": "Foto",
+    "Upload": "Hochladen",
+    "Full name": "Vorname",
+    "Language": "Sprache",
+    "Saving…": "Am speichern…",
+    "Save": "Speichern",
+    "Delete Account": "Konto löschen",
+    "You may delete your account at any time, note that this is unrecoverable": "Sie können Ihr Konto jederzeit löschen. Beachten Sie, dass dies nicht wiederherstellbar ist",
```

Comment:
German question – is it normal to have capital letters in the middle of sentences like this, or a mistake?

Reply:
Haha - i feel you. Yes, its a very german thing to do :)

The funny part is, that most foreigners have trouble writing german because of misusing of capitalisation :D